tests/integration/pilot/vm/README.md

```diff
@@ -0,0 +1,58 @@
+# Writing Tests for VMs
+
```

Comment:
Add an Overview section with a list of scenarios with who and what-for to add a VM test?

- Istio Developers to add a VM test to cover features like security, traffic management and telemetry.
- Istio Developers to add a service running on VM to excise Istio features with services other than echo.
- Istio Developers to add a VM type to run tests on a new OS image.

Reply:
Added an Overview section.